src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/messaging/MessageProcessor.java

```diff
@@ -186,16 +187,18 @@ private MessageData handleError(Throwable error, String actionDefinition) {
         }
     }
 
-    private void sendResponse(Response response, String correlationId, String key) {
-        Message message = makeMessage(response.getPayload(), correlationId);
+    private void sendResponse(Response response, String correlationId, String key, MessageCookie cookie) {
+        log.debug("GRPC speaker response on request {} with cookie {}: {}", key, cookie, response);
```

Comment:
If this method should be used only for GRPC responses maybe we need to rename it?

Reply:
Didn't get. This is GRPC speaker, what other respones it can use? Or why is should qualify more it's own methods with "grpc" term anywhere?